web/src/modules/user/p2pOrders/reducer.ts

```diff
@@ -120,6 +121,15 @@ const orderReducer = (state: P2POrdersState['order'], action: P2POrdersActions)
                 ...state,
                 success: false,
             };
+        case P2P_ORDER_UPDATE_STATUS:
+            console.log('P2P_ORDER_UPDATE_STATUS: ', action.payload);
+            return {
+                ...state,
+                data: {
+                    ...state.data,
+                    state: action.payload,
+                },
+            };
```

Comment:
please, remove this hardcoding of order status, we should handle this case when BE returns order with autocancelled state